### ISSUE TITLE
Battlenet launcher not appearing after install on Steam Deck OLED

### ISSUE THREAD

**Issue by Gnice4U** [moraroy/NonSteamLaunchers-On-Steam-Deck#201]
I have a new Steam Deak OLED.  I watched the video and the written instructions.  One thing of note is the instructions don't include the step about custom websites.  Regardless of that fact I've now tried to install Battlenet (and Battlenet alone) 6 times (uninstalling after each failed attempt).  Each time I see the Battlenet launcher install screen.  However once that is done there is no sign on the Battlenet launcher at all.  I have tried restarting immediately after and not restarting. The result is exactly the same.  The launcher is not in the Non-steam section per the instructions.

**Comment by moraroy** [comment]
@Gnice4U send the log in the downloads folder

**Comment by Gnice4U** [comment]
[NonSteamLaunchers-install.log](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/files/14241371/NonSteamLaunchers-install.log)

**Comment by Gnice4U** [comment]
Is there anything else I can provide or do to help?

**Comment by moraroy** [comment]
@Gnice4U youre shortcuts.vdf file is corrupted for some reason it looks like. You may need to delete it and let Steam make you a new one. However this will earse your whole library...Does your current steam library work correctly? Also, this erases your "library" not your actual games installed btw. The steam library is just a list of your installed applications

**Comment by Gnice4U** [comment]
> @Gnice4U youre shortcuts.vdf file is corrupted for some reason it looks like. You may need to delete it and let Steam make you a new one. However this will earse your whole library...Does your current steam library work correctly? Also, this erases your "library" not your actual games installed btw. The steam library is just a list of your installed applications

Yes the library works fine otherwise.  I haven't had any issues at all.

**Comment by moraroy** [comment]
@Gnice4U NO dont do that!!! thats not a good thing to delete that file!

**Comment by moraroy** [comment]
@Gnice4U if your library is working normally please do not delete that file. Only delete that file as a last resort.

**Comment by Gnice4U** [comment]
> @Gnice4U if your library is working normally please do not delete that file. Only delete that file as a last resort.

Ok. I will hold off.  The tool you've built looks awesome so hope a path forward becomes available at some point.

**Comment by Gnice4U** [comment]
@moraroy just wanted to let you know it's working now.  What got me thinking was the launcher was working fine otherwise and I did see there was a option to install Google Chrome when I went to the Non-steam section.  I had not chosen to install Chrome before because I already had Firefox (on the Desktop).  But since it was prompting me to install Chrome in the Non Steam section I was curious if it would work. I also thought maybe it would "force" a rebuild of the the shortcuts file.  

So installed Chrome from the Non Steam section and it showed up and worked.  So I immediately went into the desktop and installed your tool.  And BAM, it worked.

I took a bit to get logged in, but got past that and the Battlenet launcher is running on my Steam Deck.  Now I just need to install Diablo 4.  :)
